testing/tests/DevExpress.core/utils.format.tests.js

```diff
@@ -135,3 +137,17 @@ QUnit.test("percent format parsing", function(assert) {
     assert.strictEqual(parser("-10.0%"), -0.1, "parse negative float number with 1 digit");
     assert.strictEqual(parser("-10%"), null, "negative value without float part should be incorrect");
 });
+
+
+QUnit.module("number formatter");
+
+QUnit.test("float with precision formatting", function(assert) {
+    var formatter = generateNumberFormat("#.00");
+
+    assert.strictEqual(formatter(null), "0.00", "format an empty value");
```

Comment:
This is not correct. Our formatter should work similar to globalize/intl and similar to current build-in number formatter.